docs/installation/resources/example-services.yml

```diff
@@ -0,0 +1,14 @@
+# Do not commit this file. This file is portal specific.
```

Comment:
Change to: "Do not commit this file with your app code. This file is portal specific."

Reply:
Resolved